script/transform/page__alto

```diff
@@ -1 +1,2 @@
-alto__page
\ No newline at end of file
+#!/bin/bash
+./alto__page "$@" -convert-to ALTO
```

Comment:
How can this call been fixed? I tried several things but besides an absolute path nothing worked for me...

Reply:
Is this still a link? I see in the docker container now:
```
# ls -la
drwxr-xr-x    1 root     root          4096 Dec 31 00:06 .
drwxr-xr-x    1 root     root          4096 Dec 31 00:06 ..
-rw-r--r--    1 root     root           276 Dec 31 00:06 README.md
-rwxr-xr-x    1 root     root           450 Dec 31 00:10 alto__page
-rwxr-xr-x    1 root     root           453 Dec 31 00:06 gcv__hocr
lrwxrwxrwx    1 root     root            47 Dec 31 00:06 page__alto -> #!/bin/bash?./alto__page "$@" -convert-to ALTO?
lrwxrwxrwx    1 root     root            10 Dec 31 00:06 page__page2019 -> alto_
```

Reply:
Looks like it's symlinking not to the path of the file (as expected) but to the content of the file. which is insane of course. I'll have a look.

Reply:
Never mind as this PR is now superseeded. It might be related to how symlinking is working (resp not working) in Cygwin or git bash under Windows.

Reply:
I also had to remove the old installed symlink manually on Linux to get `make install` working.

Reply:
> Never mind as this PR is now superseeded

Yeah I cannot reproduce. But reopen if you encounter this again, it's an interesting bug :^)